src/components/MapView/utils.ts

```diff
@@ -97,10 +100,11 @@ export const convertToTableData = (
   });
 
   const headlessRows = groupedRowDataWithAllLabels.map(row => {
-    const total = fields.reduce((acc, o) => row[acc] + row[o]);
+    const total = fields.map(f => row[f]).reduce((a, b) => a + b);
```

Comment:
does something like `sumBy(fields, key=>row[key]);` from lodash work? Didn't look into this too far, so all good if it doesn't work :+1:

Reply:
I opted on using `vanilla` Javascript as you suggested previously. Thus reduce the bundle size

Reply:
that's fair! My stance on it is to only use lodash if it makes the code easier to read.
For example, the `map` and `reduce` functions by lodash don't serve much use for most cases since vanilla JS can do it and looks basically the same, but the use of `sumBy` here would make the code easier to read. If lodash can make the code look cleaner, I think it's worthwhile. However up to personal preference :+1: